usecases.md

```diff
@@ -150,7 +150,7 @@ impact of repeated request on the rest of the infrastructure.
 
 
 
-# End-users
+# End user
 
 * As an end user, I can understand which resources I can access and how I can request access to a resource
```

Comment:
end-user

Reply:
added note in PR description to explain why I think "end user" is actually consistent